samples/recur_no_dtend.ics

```diff
@@ -0,0 +1,55 @@
+BEGIN:VCALENDAR
+PRODID:-//Google Inc//Google Calendar 70.9054//EN
+VERSION:2.0
+CALSCALE:GREGORIAN
+METHOD:PUBLISH
+X-WR-CALNAME:http://nhlc.org/32%2C35%2C37%2C38%2C40%2C41%2C42%2C43%2C45%2C4
```

Comment:
Could you remove the data identifying any external sites, i.e. drop the `X-WR-CALNAME` and change the `UID` to something else?